sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/DefaultSectionShouldBeFirstOrLast.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+
+namespace Tests.Diagnostics
+{
+    public class Program
+    {
+        public void Foo()
+        {
+            switch (a)
+            {
+                case 0:
+                    break;
+                default: // Compliant - last section
+                    break;
+            }
+
+            switch (a)
+            {
+                default: // Compliant - first section
+                    break;
+
+                case 0:
+                    break;
+            }
+
+            switch (a)
+            {
+                case 0:
+                    break;
+                default: // Noncompliant {{Move this 'default:' case to the beginning or end of this 'switch' statement.}}
+//              ^^^^^^^^
+                    break;
+                case 1:
+                    break;
+            }
+
+            switch (a)
+            {
+            }
+
+            switch (a)
+            {
+                case 42:
+                    break;
+            }
```

Comment:
For completeness, how about a switch statement that only has a _default_ section?